consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/JobStatus.java

```diff
@@ -0,0 +1,74 @@
+package com.redhat.lightblue.migrator.consistency;
+
+/**
+ *
+ * JobStatus can be one of the following values:
+ *      NEW                - Means the Job just created and it isn't ready to be executed
+ *      READY              - Means the Job is ready to be executed but not in execution
```

Comment:
I can remove those two status (I think that what you mean) but one think that I thought to at least leave one of them is if we got the JVM crashed or something (example there is a way to record the JVM state and reproduce) and if we got that object we would know that it wasn't running. But I think it is a very uncommon use case, so I will remove NEW and READY